src/activities/discussions/GroupSectionRestrictionActionsEntity.js

```diff
@@ -0,0 +1,44 @@
+import { Actions } from '../../hypermedia-constants.js';
+import { Entity } from '../../es6/Entity.js';
+import { performSirenAction } from '../../es6/SirenAction.js';
+
+export class GroupSectionRestrictionActionsEntity extends Entity {
+	canSetToRestrictedTopic() {
+		if (!this._entity) return;
+		return !this._entity.hasActionByName(Actions.discussions.groupSectionRestrictions.setToRestrictedTopic);
```

Comment:
Should this be negated if we are checking if the action is available?

Reply:
oh yeah, good catch.